README.md

```diff
@@ -39,79 +39,96 @@ const client = new ApolloClient({
 
     /**
-     * Include the entire Apollo cache in the breadcrumb. It is not recommended to enable this
-     * option in production environment, for several reasons, see "Be careful what you include".
-     * This option is specifically useful for debugging purposes, but when applied in combination
-     * with `beforeBreadcrumb` can also be used in production.
+     * Include the fetched result (data, errors, extensions)?
+     *
+     * Be careful not to leak sensitive information or send too much data.
      */
-    includeCache: false,
+    includeResult: false | true;
 
     /**
-     * Include the operation's variables in the breadcrumb. Again, be careful what you include,
-     * or apply a filter.
+     * Include the response error?
+     *
+     * Be careful not to leak sensitive information or send too much data.
      */
-    includeVariables: false,
+    includeError: false | true;
 
     /**
-     * Include the operation's fetch result in the breadcrumb.
+     * Include the contents of the Apollo Client cache?
+     *
+     * This is mostly useful for debugging purposes and not recommended for production environments,
+     * see "Be careful what you include", unless carefully combined with `beforeBreadcrumb`.
      */
-    includeResponse: false,
+    includeCache: false | true;
 
     /**
-     * If an error is received, it can be included in the breadcrumb. Regardless of this option,
-     * the breadcrumb's type is set to error to reflect a failed operation in the Sentry UI.
+     * Include arbitrary data from the `ApolloContext`?
+     *
+     * Accepts a list of keys in dot notation, e.g. `foo.bar`. Can be useful to include extra
+     * information such as headers.
      */
-    includeError: false,
+    includeContext: false | NonEmptyArray<string>;
 
     /**
-     * Include context keys as extra data in the breadcrumb. Accepts dot notation.
-     * The data is stringified and formatted. Can be used to include headers for instance.
+     * Modify the breadcrumb right before it is sent.
+     *
+     * Can be used to add additional data from the operation or clean up included data.
+     * Very useful in combination with options like `includeVariables` and `includeContextKeys`.
      */
-    includeContextKeys: [],
-  },
-
-  /**
-   * Provide a callback function which receives an instance of this package's Operation class
-   * Only operations that pass the test are sent to Sentry. Leave undefined if you want all
-   * operations to pass. See PR #9 for more details.
-   */
-  filter: (operation) => true,
-
-  /**
-   * Provide a callback function which receives an instance of this package's OperationBreadcrumb class
-   * Use it to modify the data that is added to the breadcrumb. Leave undefined if you want all
-   * data to be included. Very useful in combination with options like includeVariables and includeContextKeys.
-   */
-  beforeBreadcrumb: (breadcrumb) => breadcrumb,
+    transform:
```

Comment:
Moved in here from a top level config, only makes sense when breadcrumbs are generally enabled.